translations/strings.yaml

```diff
@@ -215,3 +215,7 @@ customDNSSettings:
 
 networkSettings:
   multihopTitle: Multi-hop tunnel
+
+systemPreferencesSettings:
+  systemPreferencesTitle: System preferences
```

Comment:
Out of curiosity, why "preferences"? Firefox is moving towards "settings", unified across platforms (macOS, Linux, Windows).

Reply:
Thank you for the feedback! I'm not sure but maybe @chrisrooneys can add context.